insights/specs/insights_archive.py

```diff
@@ -239,7 +239,6 @@ class InsightsArchiveSpecs(Specs):
     saphostctl_getcimobject_sapinstance = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostctrl_-function_GetCIMObject_-enuminstances_SAPInstance")
     saphostexec_status = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostexec_-status")
     saphostexec_version = simple_file("insights_commands/usr.sap.hostctrl.exe.saphostexec_-version")
-    satellite_enabled_features = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_satellite_enabled_features")
```

Comment:
@huali027 rather than removing this, it would be better to change it using the path that would be created by the client for storage in the archive:
```
$ mangle "/usr/bin/curl -sk https://localhost:9090/features --connect-timeout 5"
insights_commands/curl_-sk_https_..localhost_9090.features_--connect-timeout_5
```

Reply:
@bfahr, but we agreed on that we don't add the spec "/usr/bin/curl -sk https://localhost:9090/features --connect-timeout 5" since we don't want to run it on all the hosts, right? So do you suggest we still update it to that?

Reply:
@huali027 you've correctly implemented this with a dependency on a datasource that will prevent it from running on all systems.  From an implementation standpoint, using the mangle as I suggested is the correct way to implement the archive spec.